presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveAggregationQueries.java

```diff
@@ -61,11 +61,16 @@ public void testAggregations()
 
         assertQuery("SELECT linenumber = 2 AND quantity > 10, sum(quantity / 7) FROM lineitem GROUP BY 1");
 
-        assertQuerySucceeds("SELECT approx_percentile(totalprice, 0.25), approx_percentile(totalprice, 0.5) FROM orders");
-        assertQuerySucceeds("SELECT approx_percentile(totalprice, orderkey, 0.25), approx_percentile(totalprice, orderkey, 0.5) FROM orders");
-        assertQuerySucceeds("SELECT clerk, approx_percentile(totalprice, 0.25), approx_percentile(totalprice, 0.5) FROM orders GROUP BY 1");
-        assertQuerySucceeds("SELECT approx_percentile(totalprice, 0.25, 0.005), approx_percentile(totalprice, 0.5, 0.005) FROM orders");
-        assertQuerySucceeds("SELECT approx_percentile(totalprice, orderkey, 0.25, 0.005), approx_percentile(totalprice, orderkey, 0.5, 0.005) FROM orders");
+        assertQuerySucceeds("SELECT approx_percentile(totalprice, 0.25) FROM orders");
```

Comment:
Should we also keep queries that apply approx_percentile multiple times for the same column?

Reply:
#18207 will merge them into the array version, so we will lose the coverage for non-array version if we keep multiple times for same column.

Reply:
I was thinking we can have both. Some queries would compute multiple percentiles on the same column, otherwise multiple percentiles on different columns.